src/Analyzers/Core/Analyzers/RemoveUnusedMembers/AbstractRemoveUnusedMembersDiagnosticAnalyzer.cs

```diff
@@ -486,7 +500,10 @@ private void OnSymbolEnd(SymbolAnalysisContext symbolEndContext, bool hasUnsuppo
                             break;
 
                         case IPropertySymbol property:
-                            if (property.GetMethod != null && property.SetMethod != null)
+                            if (property.GetMethod != null && property.SetMethod != null &&
+                                // We change the message only if there are no shadow 'get' accessor usages.
+                                // Otherwise the message will be confusing
```

Comment:
please place comments above hte if statement.  generally we don't lke to intersperse comments into an expression.